src/components/CreatorsList.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <creator-element
```

Comment:
https://github.com/hackathon-21-spring-02/front-end/pull/4/files#diff-cff1d0969d9c524b5458c5c1559d0e9c369c8aedb2a97332e52fe1ae48be5734R10
こっちでは `UpperCamelCase` 使ってるからそっちに合わせてほしいかも (他の部分では `kebab-case` が多いんだけどあとで直す)
```suggestion
  <CreatorElement
```